### ISSUE TITLE
Add font customization

### ISSUE THREAD

**Issue by Loirooriol** [Loirooriol/tab-counter-plus#13]
[Some users](https://addons.mozilla.org/ca/firefox/addon/tab-counter-plus/reviews/1187468/) want font customization.

I may consider adding some control for things like
 - [`font-family`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family)
 - [`font-weight`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) for having bold text
 - [`font-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style) for having italic text
 - [`font-variant-numeric`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) like `slashed-zero`, `<numeric-figure-values>` or `<numeric-spacing-values>`.

**Comment by mimecry** [comment]
this would easily be the best improvement possible to your near perfect addon. 

i miss this feature from the equivalent Pale Moon addon and would love to see it come to your work some time in the future if possible 
![image](https://user-images.githubusercontent.com/10742312/97116633-0afb2680-1731-11eb-9d8a-96528e2a65b2.png)

**Comment by Loirooriol** [comment]
Thanks for the reference, it's interesting to see what customization were offered in this legacy add-on.

 - ✔️ Font color: done
 - ❌ Font size: not possible. Firefox only gives me a tiny square, so the font size is set automatically with some heuristics so that the counter fits in there. I could maybe add some pref to affect these heuristics, but it's not as simple as just specifying the desired value.
 - 📝 Font family: reasonable, currently set to `'Segoe UI', 'DejaVu Sans', sans-serif` since it looked good to me in both Windows and Ubuntu.
 - 📝 Font weight: reasonable, though bold text is wider and thus may need to be further squeezed in order to fit
 - ✔️ Background color: done (but without semitransparency, not sure if it's worth it)
 - ❌ Outline: given the lack of space, outlines would be bad
 - ❌ Disable tooltip: doesn't seem useful, if you don't want to see the tooltip, just don't hover the icon?
 - 📝 Do not count pinned tabs: reasonable. I already have an experimental patch that lets you exclude pinned tabs, hidden tabs, unloaded tabs, or combinations of these.
 - ❓ Advanced text formatting: no idea
 - ❌ Optional text: wouldn't fit
 - ❌ Minimum width: not possible

**Comment by mimecry** [comment]
yes, all the secondary options are very much non-essential. i do hope you can find a good implementation to increase font size, because the SVG version is a little too small at the moment imo

![image](https://user-images.githubusercontent.com/10742312/97128637-74018f00-176f-11eb-87e0-fd75826d6c4c.png)

p.s. here's the description of what Advanced text formatting does in the legacy addon, i wouldn't use it but thought you might be curious to know either way

> The display label can be in simple (default) or advanced format.

> In first case, the extension shows only the number of open tabs in the window and if you want, you can
add some text before and/or after this number.

> The advanced formatting can be enabled by checking the option "Use advanced text formatting".
In this mode, there are ten keys available, which allows you to compose your own display label, showing
the information you want to be shown.

**Comment by Loirooriol** [comment]
> i do hope you can find a good implementation to increase font size, because the SVG version is a little too small

I can set whatever `font-size` I want. The problem is that the icon must be 16x16 pixels, which is tiny (when you hover the button it seems to be 28x28, but 12 pixels are just padding). So if I increase the font-size (which I would want to do), then the number would be clipped :(

> here's the description of what Advanced text formatting does in the legacy addon

Thanks. It's not possible to have that since the text wouldn't fit.

> is it just me or has the counter in all the webextension badges been decreased in size?

Still 16x16 pixels for icons. For text badges, there is `font-size: 10px` for windows and linux, and `font-size: 9px` for osx. No recent change.

**Comment by mimecry** [comment]
that's unfortunate. i guess the only solution is to enable font-weight so you can add a bit of heft to the text to make it look not as skinny next to the other icons.

oh, i meant the number counter on the badge itself. i went back and confirmed it to be the case, although the change seem to have happened quite a while ago. the upper one is from Waterfox which i was using until recently (based on FF 68.0) and the second one is from Icecat (based on FF 78.4 ESR)

![image](https://user-images.githubusercontent.com/10742312/97247955-467e1980-1833-11eb-98e6-0611d1b454f8.png)
